src/olympia/devhub/tests/test_tasks.py

```diff
@@ -579,9 +579,9 @@ def test_amo_validator_addons_linter_error(self):
         self.create_switch('addons-linter')
 
         # This test assumes that `amo-validator` doesn't correctly
-        # validate a missing id in manifest.json
+        # validate a invalid id in manifest.json
         result = tasks.validate_file_path(
-            get_addon_file('invalid_webextension_missing_id.xpi'),
+            get_addon_file('invalid_webextension_invalid_id.xpi'),
```

Comment:
The linter does everything correct. This test is based on the idea that `addons-linter` actually validates the id for a webextension and `amo-validator` doesn't so I'm using that to not just test "did I call this?" but also to verify that we get something expected back from the linter.